src/components/layer-controller/LayerControllerSubscriber.js

```diff
@@ -7,12 +7,23 @@ import {
   createGenerateClassName,
 } from '@material-ui/core/styles';
 import { createZarrLoader, createOMETiffLoader } from '@hubmap/vitessce-image-viewer';
-
 import TitleInfo from '../TitleInfo';
-import LayerController from './LayerController';
+import RasterLayerController from './RasterLayerController';
+import NonRasterLayerController from './NonRasterLayerController';
 import ImageAddButton from './ImageAddButton';
 import {
-  RASTER_ADD, LAYER_REMOVE, CLEAR_PLEASE_WAIT, METADATA_REMOVE, LAYER_ADD, METADATA_ADD,
+  RASTER_ADD,
+  LAYER_REMOVE,
+  CLEAR_PLEASE_WAIT,
+  METADATA_REMOVE,
+  LAYER_ADD,
+  METADATA_ADD,
+  CELLS_OPACITY,
+  CELLS_ADD,
+  CELLS_ON,
+  MOLECULES_ADD,
+  MOLECULES_OPACITY,
+  MOLECULES_ON,
```

Comment:
```suggestion
  CELLS_SET_OPACITY,
  CELLS_ADD,
  CELLS_TURN_ON,
  MOLECULES_ADD,
  MOLECULES_SET_OPACITY,
  MOLECULES_TURN_ON,
```
Slight preference for making events clearly verbs.
(same applies for the actual values.)